osx-compile.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#set -x #Debug
+set -e #exit on failure
+
+#Prepare for ros2
+cd /tmp
+wget https://github.com/ros2/ros2/releases/download/release-crystal-20190214/ros2-crystal-20190214-macos-amd64.tar.bz2
+tar jxf ros2-crystal-20190214-macos-amd64.tar.bz2
+# Remove tf2_eigen
+find ros2-osx/ -name tf2_eigen | xargs rm -rf
+
+source ros2-osx/setup.bash
+mkdir -p /tmp/ros2_ws/src
+cp -r /tmp/moveit2 /tmp/ros2_ws/src
+cd /tmp/ros2_ws && wget https://raw.githubusercontent.com/AcutronicRobotics/moveit2/master/external-repos.repos
+vcs import src < external-repos.repos
+export OPENSSL_ROOT_DIR="/usr/local/opt/openssl"
+#Ignore packages
+touch src/image_common/camera_calibration_parsers/COLCON_IGNORE
```

Comment:
Can you elaborate on why `camera_calibration_parsers` shouldn't be built in a comment? If this is an OSX issue then we will want to know that we can delete this line when the package is fully ported

Reply:
Only because for now is not needed, and because most of the packages on this repo have COLCON_IGNORE. We can keep this as is till have correctly ported to ROS2.